localgov_skeleton.theme

```diff
@@ -4,3 +4,65 @@
  * @file
  * LocalGov Drupal skeleton theme file.
  */
+
+/**
+ * Implements hook_preprocess_HOOK for form_element.
+ */
+function localgov_skeleton_preprocess_form_element(&$variables) {
+  $element =& $variables['element'];
```

Comment:
Minor issue, but it doesn't look like assignment by reference is needed here because `$element` is never changed.

Reply:
@ekes any comment on this?

Reply:
I've never wondered about this does the PHP engine optimise out copies of arrays that aren't altered? Does it matter? Can it be huge enough to be an issue.

Reply:
It doesn't really matter. Just flagged it because there are other hooks that contain a normal assignment (`$element = $variables['element'];`). Just pointing it out for a consistency and in case there was a particular reason for using assignment by reference in this particular case.